app/models/income.rb

```diff
@@ -1,3 +1,4 @@
 class Income < ActiveRecord::Base
   belongs_to :game
+  belongs_to :team
```

Comment:
This is really a note for myself to do later but it is useful for you to know:

Since eventually Team gets tied to a Game.  We can update Incomes using delegate method to get Games through Team.

Therefore it would be: 
```
belongs_to :team
delegate :game, to: :team
```